python/grass/jupyter/utils.py

```diff
@@ -69,9 +71,16 @@ def reproject_region(region, from_proj, to_proj):
         raise RuntimeError(
             _("Encountered error while running m.proj: {}").format(stderr)
         )
-    enws = gs.decode(proj_output).split(os.linesep)
-    elon, nlat, unused = enws[0].split(" ")
-    wlon, slat, unused = enws[1].split(" ")
+    enws = gs.decode(proj_output).splitlines()
+    # get the largest bbox
+    n = []
+    e = []
+    for each in enws:
```

Comment:
Although I like the `for each` idea, `for row in data` seem more readable to beginners.